tests/phpunit/CRM/Pledge/BAO/PledgeTest.php

```diff
@@ -149,7 +149,7 @@ public function testGetPledgeStartDate() {
 
     // Try with fixed date
     $params = array(
-      'pledge_start_date' =>  serialize(array('contribution_date' => '2016-06-10')),
+      'pledge_start_date' => serialize(array('contribution_date' => '2016-06-10')),
```

Comment:
I think as preference we are switching to json_encode now rather than serialize @colemanw is that your feeling ?